packages/bee-q/src/components/tab/scss/bq-tab.scss

```diff
@@ -0,0 +1,43 @@
+/* -------------------------------------------------------------------------- */
+/*                                Tab styles                               */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-tab-variables';
+
+:host {
+  @apply inline-block;
+}
+
+.bq-tab {
+  @apply relative flex cursor-pointer items-center justify-center disabled:cursor-not-allowed;
+  @apply text-m font-medium leading-large [&:not([disabled])]:hover:text-text-brand-hover;
+  @apply border-0 bg-transparent focus-visible:focus focus-visible:rounded-s;
+
+  &--small {
+    @apply py-[var(--bq-tab--pading-vertical-small)] px-[var(--bq-tab--pading-horizontal-small)];
+
+    ::slotted(bq-icon) {
+      --bq-icon--size: var(--bq-tab--icon-size-small) !important;
+    }
+  }
+
+  &--medium {
+    @apply py-[var(--bq-tab--pading-vertical-medium)] px-[var(--bq-tab--pading-horizontal-medium)];
+
+    ::slotted(bq-icon) {
+      --bq-icon--size: var(--bq-tab--icon-size-medium) !important;
+    }
+  }
+
+  &--large {
+    @apply py-[var(--bq-tab--pading-vertical-large)] px-[var(--bq-tab--pading-horizontal-large)] text-l;
+
+    ::slotted(bq-icon) {
+      --bq-icon--size: var(--bq-tab--icon-size-large) !important;
+    }
+  }
+}
+
+.bq-tab__underline {
+  @apply w-full border-x-0 border-t-0;
```

Comment:
Using a `margin-top: 2px` on the underline will make the focus to looks better I guess:

![CleanShot 2023-03-22 at 10 11 19](https://user-images.githubusercontent.com/328492/226839659-6542d055-f4fe-483d-93b5-f8be4a9a47ec.png)

Reply:
Done, thanks.